frontend/static/federation.json

```diff
@@ -142,6 +142,52 @@
     "mainnetNodesPubkeys": ["02c5b5972b05fba2cd2c2d9269a47bc478f73fae0f248a85cb1e5af60a07c1919d"],
     "testnetNodesPubkeys": ["032b698c8143f293d138c0926594f11d119194ddedb513f63a944d14c094d0e54a"]
   },
+  "overthemoon": {
+    "longAlias": "Over the moon",
+    "shortAlias": "overthemoon",
```

Comment:
`"shortAlias"` can be changed to `"moon"`, everything else looks good. Thank you so much!